RockWeb/Themes/NewSpring/assets/Lava/LAYOUT/network-resource-collection.lava

```diff
@@ -67,71 +64,33 @@
-            SELECT ROW_NUMBER() OVER (ORDER BY cci.StartDateTime) - 1 'Index', cci.Id, ccis.Slug
-            FROM ContentChannelItemAssociation ccia
-            JOIN ContentChannelItem cci
-            ON ccia.ChildContentChannelItemId = cci.Id
-            JOIN ContentChannelItemSlug ccis
-            ON ccis.ContentChannelItemId = cci.Id
-            WHERE ccia.ContentChannelItemId = '{{ cciid }}'
-            {% if ccids and ccids != empty %}AND cci.ContentChannelId IN ({{ ccids }}){% endif %}
-            ORDER BY cci.StartDateTime
+					{% if childItem.ChannelName contains 'Sermon' or childItem.ChannelName contains 'Lesson' %}
+                        {% assign type = child.Index | Prepend:'Week '  %}
+                    {% else %}
+                        {% assign type = childItem.ChannelName | Split:' - ' | Index:1 | Singularize %}
+                    {% endif %}
 
-        {% endif %}
-    {% endsql %}
 
-    {% assign parentType = Item.ContentChannel.Name | Split:' - ' | Last | Singularize %}
-    {% assign type = Item.ChildItems | First %}
-    {% assign childType = type.ChildContentChannelItem.ContentChannel.Name | Split:' - ' | Last | Singularize %}
 
-    {[ swiper title:'{{ childType | Pluralize }} From This {{ parentType | Singularize }}' ]}
-        {% for result in results %}[[ item ]]
-            {% contentchannelitem id:'{{ result.Id }}' iterator:'children' %}
-                {% for child in children %}
-                    {% assign id = child.Id %}
-                    {% assign cciid = child.Id %}
-                    {% assign type = result.Index | Plus:1 | Prepend:'Week ' %}
-                    {% assign video = child | Attribute:'Video','RawValue' %}
-                    {% assign parentimage = child.ParentItems | First | Property:'ContentChannelItem','Object' | Attribute:'ImageLandscape','Url' %}
-                    {% assign childimage = child | Attribute:'ImageLandscape','Url' %}
-                    {% capture imageurl %}
-                        {% if childimage and childimage != empty %}
-                            {{ childimage }}
-                        {% elseif video and video != empty %}
-                            {[ getImageFromVideoId id:'{{ video }}' resolution:'1000x500' ]}
-                        {% elseif parentimage and parentimage != empty %}
-                            {{ parentimage }}
-                        {% endif %}
-                    {% endcapture %}
-                    {% assign imageurl = imageurl | Trim %}
-                    {% assign childVideosString = child | Attribute:'Videos' %}
-                    {% assign childVideosObject = childVideosString | FromJSON %}
-                    {% assign childDownloadsString = child | Attribute:'Downloads' %}
-                    {% assign childDownloadsObject = childDownloadsString | FromJSON %}
+                    {% assign video = childItem.Video %}
+                    {% assign childimage = childItem.ImageLandscape %}
```

Comment:
Are these used? I see where it is passing `content` to the `card` down below (which is captured right after this), but I don't see it passing these.

Reply:
Removed the video one, I changed `childimage` to `imageurl` which is what the card was being passed below.